packages/remix-dev/compiler/loaders.ts

```diff
@@ -28,6 +28,7 @@ export const loaders: { [ext: string]: esbuild.Loader } = {
   ".ttf": "file",
   ".wav": "file",
   ".webm": "file",
+  ".webmanifest": "json",
```

Comment:
Hold up — this should import as a `"file"` instead in order to solve the use case requested in #2954 — see my comment here: https://github.com/remix-run/remix/issues/2954#issuecomment-1117999207